### DISCUSSION TITLE
Support for double bracket wiki link notation [[

### DISCUSSION THREAD

**Discussion by almereyda** [outline/outline#3614] [Ideas]
Nowadays that more and more Markdown-based "digital gardens" support a wiki link notation with square brackets `[[` encapsulating a page title `]]`, and that even WordPress six has it:

> - Type two open brackets `[[` to quickly access a list of recent posts and pages.
> https://wordpress.org/news/2022/05/arturo/

we could imagine the editor to recognise writing `[[` as an intent to link to an internal page, and display the respective pop up immediately, skipping a typing path via `Ctrl` + `k` or `/` `l` `i` `n` `Return`.

Parsing wiki style-links could also become interesting when considering an import from such (Markdown) digital gardens.

Auto-Complete of slash commands in course of searching for a place to link to is also mentioned in

- https://github.com/outline/outline/discussions/3484

**Comment by Limezy** [comment]
https://github.com/outline/outline/discussions/2754 related

**Reply by almereyda** [reply]
Perfect that searching the discussions for `[[` didn't reveal it.
